test/functional/Java8andUp/playlist.xml

```diff
@@ -1959,5 +1959,26 @@
 			<subset>SE110</subset>
 		</subsets>
 	</test>
-
+	<test>
+                <testCaseName>reflect_getDeclaredFields</testCaseName>
```

Comment:
If there is no special requirement (i.e., variations, platforms, etc), we can take advantage of `generalSanityTest`. This means no playlist.xml modification is needed. All we need to do is to add `<class name="org.openj9.test.reflect.GetDeclaredFieldsTests" />` in `<test name="generalTest">` in Java8andUp/testng.xml. And the test will run as part of `generalSanityTest`.